priv/repo/migrations/20240326155113_drop_jsonb_gin_index_from_dataclips.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Lightning.Repo.Migrations.DropJsonbGinIndexFromDataclips do
+  use Ecto.Migration
+
+  def up do
+    drop index(:dataclips, [:body])
+  end
+
+  def down do
+    create index(:dataclips, [:body], concurrently: true, using: :gin)
+  end
```

Comment:
Probably won't be used but for the `down` we would need:
```
@disable_ddl_transaction true
@disable_migration_lock true
```

Reply:
@taylordowns2000 @jyeshe is correct, can you put those in - however unlikely their use will be.